go/enclave/nodetype/sequencer.go

```diff
@@ -234,6 +242,8 @@ func (s *sequencer) createNewHeadBatch(l1HeadBlock *common.L1Block, skipBatchIfE
 		}
 	}
 
+	sort.Sort(SortableTransactions(transactions))
```

Comment:
what's up with this?

Reply:
Running the simulation it seemed like transactions werent ordered by nonces, but I'm not sure its necessary. The geth mempool is just a bit weird.

Reply:
Umm. I'd say this deserves a bit of investigation. Maybe instead of sorting, check that what the mempool produces is sorted by nonce. And if not, we should investigate why not. It's probably some misconfiguration